README.md

```diff
@@ -158,7 +164,7 @@ Taking this into account, you have to set the fuzzer's output directories to /im
 $ odfuzz <SERVICE_URL> -l logs_directory -s stats_directory -r restrictions/basic.yaml -a -f -p
 ```
 
-The option **-a** enables fuzzer to send asynchronous requests. A default number of the asynchronous requests can be changed. To do so, navigate to the file *config/fuzzer/config.yaml* and modify value *async_requests_num*. Notice that some services do not support more than 10 asynchronous requests at the same time.
+The option **-a** enables the fuzzer to send concurrently multiple requests. The default number of the asynchronous requests can be changed. To do so, use the environment variable `ODFUZZ_ASYNC_REQUESTS_NUM`. Notice that some services do not support handling of more than 10 asynchronous requests at the same time.
```

Comment:
The part about "not support handling more than 10 asynch" is I believe actually not true.

The number 10 is default value, based on idea of not overloading our sap test servers with multiple odfuzz instances in the wild. However I think user can provide any number in the env variable `export ODFUZZ_ASYNC_REQUESTS_NUM=10` and its up to gevent and target server to handle it properly.